nx.json

```diff
@@ -54,6 +54,22 @@
       "!{projectRoot}/src/test-setup.[jt]s"
     ]
   },
+  "release": {
+    "projectsRelationship": "independent",
+    "groups": {
+      "northstar": {
+        "releaseTagPattern": "@fluentui/react-northstar_v{version}",
+        "projectsRelationship": "fixed",
+        "projects": ["packages/fluentui/*"],
+        "version": {
```

Comment:
You probably want to move the version config up to the top level so that it applies to all groups